Source/Error.swift

```diff
@@ -18,9 +17,30 @@ public extension Moya.Error {
         case .jsonMapping(let response): return response
         case .stringMapping(let response): return response
         case .statusCode(let response): return response
-        case .data(let response): return response
         case .underlying: return nil
         case .requestMapping: return nil
         }
     }
 }
+
+// MARK: - Error Descriptions
+
+extension Moya.Error: LocalizedError {
+    public var errorDescription: String? {
+        switch self {
+        case .imageMapping:
+            return "Failed to map data to an Image"
+        case .jsonMapping:
+            return "Failed to map data to a JSON"
+        case .stringMapping:
+            return "Failed to map data to a String"
+        case .statusCode:
+            return "Status code didn't fall within the given range"
+        case .requestMapping:
+            return "Failed to map Endpoint to a URLRequest"
```

Comment:
Sorry for the little nitpick, but would you mind adding periods to these descriptions? Otherwise, they look great.